dev/com.ibm.websphere.security/src/com/ibm/websphere/security/auth/ValidationFailedException.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.security.auth;
+
+/**
+ *
+ */
+public class ValidationFailedException extends InvalidTokenException {
```

Comment:
If this is a public API then the source should have some good javadoc

Reply:
done